### ISSUE TITLE
New challenge

### ISSUE THREAD

**Issue by unixpickle** [msemple1111/kahoot-hack#6]
I assume that Kahoot's latest challenge has broken this program (although I have not tested it). The new challenge is a bunch of JavaScript code, which can't be run with Python's eval().

Here is an example of a new challenge:

```
decode('mj1Zc77uYC6jYJ3lMUJu0STyiXuGgYmKtO34SSq2y3ztInEbvKvmcFKql4PqczIcMVnc3KtaVW8F32A8hvsURq7z0TVUqEZLNYHN'); function decode(message) {return _.replace(message, /./g, function(char, position) {return String.fromCharCode((((char.charCodeAt(0) * position) + 12) % 77) + 48)});}
```

**Comment by Egoscio** [comment]
Extract the string and evaluate it with this function I made with my limited Python experience.

```python
def decode(message):
	flist = []
	for position, char in enumerate(list(message)):
		flist.append(chr((((ord(char) * position) + 12) % 77) + 48))
	return "".join(flist)

print(decode("mj1Zc77uYC6jYJ3lMUJu0STyiXuGgYmKtO34SSq2y3ztInEbvKvmcFKql4PqczIcMVnc3KtaVW8F32A8hvsURq7z0TVUqEZLNYHN"))
```

Merry Christmas! Ho ho ho 🎅

**Comment by unixpickle** [comment]
The first parameter can change, but the other two do not appear to change. Still, anything goes with these challenges; they may easily change in the future, although it's much more likely that the code will just change drastically in general.

**Comment by Egoscio** [comment]
Well, if all else fails, [PyExecJS](https://pypi.python.org/pypi/PyExecJS) is an option. I'm personally interested in making my own Node port when I get the time, as Python isn't my bread & butter.

**Comment by unixpickle** [comment]
@Egoscio right, at this point Kahoot has basically made Node the best option. That's why my [kahoot-hack](https://github.com/unixpickle/kahoot-hack) invokes a Node web application to solve the challenges.

**Comment by msemple1111** [comment]
fixed #efcc636, using the safeval.pw service to evaluate it.
